sqlstore/src/main/resources/db/migration/V8502__changeloggables.sql

```diff
@@ -0,0 +1,43 @@
+SELECT userId INTO @admin FROM User WHERE loginName = 'admin';
```

Comment:
`--Start/EndNoTest`?

Reply:
Yep. Also need to finish modifying test data scripts to include the new fields to fix some tests